src/protocol/deserialize_auto.cpp

```diff
@@ -2576,14 +2576,19 @@ namespace OpcUa
 
 
     template<>
+    void DataDeserializer::Deserialize<CallRequestParameters>(CallRequestParameters& data)
+    {
+        DeserializeContainer(*this, data.MethodsToCall);
+    }
+
+    template<>
     void DataDeserializer::Deserialize<CallRequest>(CallRequest& data)
```

Comment:
There is always space for improvement. ;-) ... even more than 10 files. -> 11 ;-) Okay. 9 without considering the server/client examples. :-)